docs/src/internals.md

```diff
@@ -401,23 +403,29 @@ poly(x, n) = x^n
 const POLY_CONTEXT = Any
 
 # struct for behavior
-struct PolyTerm <: AbstractTerm
-    term::ContinuousTerm
-    deg::Int
+struct PolyTerm{T,D} <: AbstractTerm
+    term::T
+    deg::D
 end
 
 Base.show(io::IO, p::PolyTerm) = print(io, "poly($(p.term), $(p.deg))")
 
-function StatsModels.apply_schema(t::FunctionTerm{typeof(poly)},
+# first pass: create PolyTerm
+StatsModels.apply_schema(t::FunctionTerm{typeof(poly)},
```

Comment:
Because this is split onto so many lines.
I think it would be much easier to read if it stated with `function`
and ended with `end`